src/library/autodj/dlgautodj.ui

```diff
@@ -138,6 +138,26 @@ If no track sources are configured, the track is added from the library instead.
        </property>
       </widget>
      </item>
+     <item>
+      <widget class="QLabel" name="fadeModeLabel">
+       <property name="text">
+        <string>Fade mode:</string>
```

Comment:
I do not understand your concern about this. I have tested every skin at its minimum width and everything fits as it is.

Reply:
The splitter is locked here: 
![grafik](https://user-images.githubusercontent.com/1777442/64083354-07336600-cd1f-11e9-90c5-df6f46634422.png)

Reply:
I can't reproduce that:
![image](https://user-images.githubusercontent.com/9455094/64083421-86a44380-cce5-11e9-9609-3a645dfa5da3.png)

Reply:
Oh, I see. Text in the left library pane only gets cut off when both the cover art is showing and tracks are selected in the AutoDJ playlist so their time is shown above the track table.

Reply:
I think this can be solved by simply making the fade mode combobox take less horizontal space. I do not understand why it is currently occupying more space than its widest option requires. Also, the names we discussed below will be shorter.

Reply:
I think you are right that the "Fade mode" label is unnecessary. I think the options in the combobox explain what it is well enough.